package.json

```diff
@@ -65,7 +65,8 @@
     "inline-style-prefixer": "^3.0.6",
     "normalize-css-color": "^1.0.2",
     "prop-types": "^15.5.10",
-    "react-timer-mixin": "^0.13.3"
+    "react-timer-mixin": "^0.13.3",
+    "@ecliptic/react-art": "0.15.4-fork.1"
```

Comment:
Check out this issue reactjs/react-art#113 
The npm package no longer worked with latest react version, unfortunately no one is publishing it :(